javascript/javascript3/week3/homework/index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8"/>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+    <title>Javascript3 HomeWork/Week3</title>
+    <link rel="stylesheet" href="./style.css"/>
+</head>
+<body>
+<div class="container">
+    <div class="input-container">
+        <input type="text" class="input-url" placeholder="Enter URL">
+        <button class="list-all-screenshots">List of all screenshots</button>
```

Comment:
The naming of the list-all-screenshots, render-screenshot, and render-list-of-screenshots could be confusing. You could for example make a reference to the functionality of the element you are selecting. For example fetch-all-screenshots-button, or render-screenshot-container.